test/CMakeLists.txt

```diff
@@ -38,19 +43,22 @@ add_subdirectory(bloom)
 add_subdirectory(checkpoint)
 add_subdirectory(cursor_order)
 add_subdirectory(fops)
-add_subdirectory(format)
 add_subdirectory(huge)
 add_subdirectory(manydbs)
-add_subdirectory(packing)
-add_subdirectory(readonly)
-add_subdirectory(salvage)
-add_subdirectory(syscall)
-add_subdirectory(thread)
-add_subdirectory(cppsuite)
 add_subdirectory(csuite)
 
-if("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_C_COMPILER_ID}" STREQUAL "AppleClang")
-    # The fuzz test only works with llvm compilers. Restrict compilation of this test to only
-    # when using Clang.
-    add_subdirectory(fuzz)
+if(WT_POSIX)
+    add_subdirectory(format)
+    add_subdirectory(packing)
+    add_subdirectory(readonly)
+    add_subdirectory(salvage)
+    add_subdirectory(syscall)
+    add_subdirectory(thread)
+    add_subdirectory(cppsuite)
+
+    if("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_C_COMPILER_ID}" STREQUAL "AppleClang")
+        # The fuzz test only works with llvm compilers. Restrict compilation of this test to only
+        # when using Clang.
+        add_subdirectory(fuzz)
```

Comment:
I think LibFuzzer is supported on Windows builds of Clang.

Reply:
Though we discussed offline that we currently don't have Windows build support with clang, I was mostly being defensive since I couldn't actually test it on Windows. Though thinking about it, I can still move the check outside the `WT_POSIX` statement in case we eventually explore a clang windows build?